src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -144,10 +145,16 @@ export const selectPortfolioAccountMetadata = createDeepEqualOutputSelector(
   accountMetadata => accountMetadata,
 )
 
+export const selectPortfolioAccountMetadataByAccountId = createSelector(
+  selectPortfolioAccountMetadata,
+  selectAccountIdParamFromFilter,
+  (accountMetadata, accountId): AccountMetadata => accountMetadata[accountId],
+)
+
 export const selectBIP44ParamsByAccountId = createSelector(
   selectPortfolioAccountMetadata,
   selectAccountIdParamFromFilter,
-  (accountMetadata, accountId): BIP44Params | undefined => accountMetadata[accountId]?.bip44Params,
+  (accountMetadata, accountId): BIP44Params => accountMetadata[accountId]?.bip44Params,
```

Comment:
this is a rare case where we can safely do this, nice quality of live improvement 👍